libs/sdk-ui-tests-e2e/cypress/tools/drillToModal.ts

```diff
@@ -73,12 +73,36 @@ export class DrillToModal {
             .find(".s-drill-down")
             .should("be.visible")
             .contains(`${attr}`)
-            .click();
+            .click({ force: true });
         return this;
     }
 
     hasTitleHeader(title: string) {
         this.getTitleElement().should("have.text", title);
         return this;
     }
+
+    getTooltipContentsModal(index: number) {
+        this.getElement().find(".highcharts-data-label text, .highcharts-series rect").eq(index).realHover();
+
+        cy.wait(500);
+        const result: string[] = [];
+        cy.get(".gd-viz-tooltip-item .gd-viz-tooltip-title:visible").each(($li) => {
+            return result.push($li.text());
+        });
+        return cy.wrap(result);
+    }
+
+    hasTooltipContentsModal(index: number, tooltip: string[]) {
```

Comment:
no need this fuction, you can use getChart() then reuse the function in chart.ts